src/schemas/map.json

```diff
@@ -86,21 +86,49 @@
     },
     "ItemBatcher":{
       "type": "object",
-      "oneOf": [
+      "properties": {
+        "MaxItemsPerBatch": {
+          "type": "number",
+          "minimum": 0
+        },
+        "MaxItemsPerBatchPath": {
+          "$ref": "paths.json#/definitions/asl_ref_path"
+        },
+        "MaxInputBytesPerBatch": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 262144
+        },
+        "MaxInputBytesPerBatchPath": {
+          "$ref": "paths.json#/definitions/asl_ref_path"
+        },
+        "BatchInput": {
+          "$ref": "paths.json#/definitions/asl_payload_template"
+        }
+      },
+      "anyOf": [
         {
-          "type": "object",
-          "properties": {
-            "MaxItemsPerBatch": {
-              "type": "number"
-            }
+          "required": ["MaxItemsPerBatch"]
+        },
+        {
+          "required": ["MaxItemsPerBatchPath"]
+        },
+        {
+          "required": ["MaxInputBytesPerBatch"]
+        },
+        {
+          "required": ["MaxInputBytesPerBatchPath"]
+        }
+      ],
+      "allOf": [
```

Comment:
Prevents specification of both static and path-based properties together. I'm not certain if this is the best way to do it - the error message is not particularly helpful if you violate this rule: `"Message": "/States/Map/ItemBatcher is invalid. must NOT be valid",`